NEWS.md

```diff
@@ -2,6 +2,7 @@
 
 * Status date set on newly created instance records (MODINVSTOR-509)
 * Item status date is set for newly created item records (MODINVSTOR-508)
+* source property in holdingsrecordssource.json has a type property.  (MODINVSTOR-619)
```

Comment:
I don't think this change is important enough to be in the news

Reply:
Removed.